src/shared/components/query/QueryStore.ts

```diff
@@ -1110,6 +1114,64 @@ export class QueryStore
                 return [];
             }
     }
+    
+    readonly volcanoPlotTableData = remoteData<Geneset[]>({
+        invoke: async()=>{
+            const tableData: Geneset[] = [];
+            const hierarchyData = await getHierarchyData(
+                    this.getFilteredProfiles("GENESET_SCORE")[0].molecularProfileId, 
+                    Number(this.volcanoPlotSelectedPercentile.value),
+                    0, 1, this.defaultSelectedSampleListId);
+            for (const node of hierarchyData) {
+                if (_.has(node, 'genesets')) {
+                    for (const geneset of node.genesets) {
+                        tableData.push({
+                            genesetId: geneset.genesetId,
+                            name: geneset.genesetId,
+                            description : geneset.description,
+                            representativeScore : geneset.representativeScore,
+                            representativePvalue : geneset.representativePvalue,
+                            refLink : geneset.refLink,
+                        });
+                    }
+                }
+            }
+            return tableData;
+        }
+    });
+    
+    readonly maxYVolcanoPlot = remoteData<number>({
+        await: () => [this.volcanoPlotTableData],
+        invoke: () => {
+            let maxY = 1;
+            const tableData = this.volcanoPlotTableData.result!;
+            tableData.map(({representativePvalue}) => {

```

Comment:
seems like maybe this "map" is not necessary?  representativePvalue doesn't seem to be used anywhere.